legal-api/src/legal_api/services/involuntary_dissolution.py

```diff
@@ -149,6 +150,12 @@ def _get_businesses_eligible_query(eligibility_filters: EligibilityFilters = Eli
                     _is_limited_restored(),
                     _is_xpro_from_nwpta()
                 )
+            ).\
+            order_by(
+                no_transition_filed_after_restoration.desc(),
+                _has_no_transition_filed_after_restoration().asc(),
+                specific_filing_overdue.desc(),
+                _has_specific_filing_overdue().asc()
```

Comment:
does this `order_by` code break if any of the clauses returns `None`?

Reply:
no_transition_filed_after_restoration and specific_filing_overdue will only return true or false. 

_has_no_transition_filed_after_restoration() might return None when we can't find the expected result. For this case, SQLAlchemy will handle None values by putting them at the end of the result (cause we use ascending order).

The result after ordering would be:
 1. rows sorted based on no_transition_filed_after_restoration. True values appear before False values(In descending order).
 2. The rows with the same no_transition_filed_after_restoration value would be prioritized in ascending order with the Non-None value, and None values will be placed after them.
 3. specific_filing_overdue and _has_specific_filing_overdue have the same ordering logic.
<img width="1859" alt="image" src="https://github.com/bcgov/lear/assets/168996250/13cdd4ca-c804-48f1-ae07-d612615a8218">

Reply:
great, thanks for the explanation!